.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "loopback",
+  "parserOptions": {
+    "sourceType": "module"
+  }
```

Comment:
Do you need lines 3-5? I don't see any code here using ES6 import/export. I suggest removing until you do have something using it.